docs/content/latest/develop/query-1-performance/explain-analyze.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Optimizing Queries with EXPLAIN and EXPLAIN ANALYZE
+linkTitle: Optimizing Queries with EXPLAIN and EXPLAIN ANALYZE
+description: Query optimization with EXPLAIN and EXPLAIN ANALYZE
+aliases:
+  - /develop/query-1-performance/explain-analyze/
+headerTitle: Optimizing Queries with EXPLAIN and EXPLAIN ANALYZE
+image: /images/section_icons/index/develop.png
+menu:
+  latest:
+    identifier: explain-analyze
+    parent: query-1-performance
+    weight: 566
+isTocNested: true
+showAsideToc: true
+---
+
+This section describes how to optimize queries using YSQL's `EXPLAIN` and `EXPLAIN ANALYZE` statements.
+
+## The EXPLAIN Statement and Its Options
+
+For each query that YSQL receives, it creates an execution plan which you can access using the `EXPLAIN` statement. The plan not only estimates the initial cost before the first row is returned, but also provides an estimate of the total execution cost for the whole result set.
+
+You can use the `EXPLAIN` statement in conjunction with `SELECT`, `DELETE`, `INSERT`, `REPLACE`, and `UPDATE` statements. To define the `EXPLAIN` statement, use the following syntax:
```

Comment:
This phrasing isn't common...

To define the `EXPLAIN` statement, use the following syntax:

Maybe something like:


The `EXPLAIN` statement has the following syntax:

Reply:
Fixed